core/iwasm/compilation/aot_compiler.c

```diff
@@ -318,11 +318,11 @@ aot_gen_commit_values(AOTCompFrame *frame)
     LLVMValueRef value;
     uint32 n;
 
-    for (p = frame->lp; p < frame->sp; p++) {
+    /* First, commit reference flags, ignore local(params + locals) ref flags */
+    for (p = frame->lp + frame->max_local_cell_num; p < frame->sp; p++) {
```

Comment:
Should not ignore local ref flags for JIT? Not sure why JIT's CI runs good.
For JIT, should be `p = frame->lp`.